src/components/dialogs/payment.vue

```diff
@@ -12,25 +12,24 @@
               </v-btn>
             </v-card-title>
 
-            <v-card-text class="copy-normal pt-0">
+            <v-card-text class="copy-normal pt-5">
               <StaffPayment
-                :doValidate="validateStaffPayment"
+                ref="staffPaymentComponent"
```

Comment:
I needed to make "validate" a state instead of a readonly prop, so I use this to call a method in this component.